### PR TITLE
UI Modernization

### PR DESCRIPTION
**Pull request by Calcyfer**
## Description
An update for the UI, using an update of the existing material design light library.

The settings have been moved to the navbar at the top.

All functionality has been tested and is working.

### Host list
hostlist

### A modal
modal

### Game list
gamelist

### CODE REVIEW

**Comment by Calcyfer** [comment]
@cgutman or @raidancampbell Could any of you review this, please?

**Comment by ShadowBullet** [comment]
Wow! That looks great! I would love to have this new design! @raidancampbell @cgutman please approve this pull request!

**Comment by ShadowBullet** [comment]
I just built the app by forking your repo, all is fantastic, however, there is one minor little thing that might need to be cleaned up. When streaming on a 16:10 screen (On my MacBook for example) instead of the bars at the top being black, they are the background color of the application. Is there a way that you may be able to modify this? It is rather distracting. I have attached an image to describe what I am experiencing.
screen shot 2017-05-01 at 12 11 59 pm

**Comment by raidancampbell** [comment]
This is definitely something I'm interested in merging. I need time time to
get my computer set up again, but this looks amazing! I'll do some complete
testing to verify it works, then report back.

If I don't reply within a week, ping me again.

**Comment by ShadowBullet** [comment]
@Calcyfer It looks like if we change the background color of when it is establishing the connection to black, then we will have the black bars instead of grey bars as that seems to be the consistent color of establishing the connection and running the connection.

**Comment by Calcyfer** [comment]
@ShadowBullet I'll have a look at it and update the PR, thanks for testing!

**Comment by Calcyfer** [comment]
@ShadowBullet I updated the PR with a change that should fix the background color.

**Comment by mattmeier612** [comment]
@Calcyfer: This looks really great.  A small issue I ran into when I built this repo was that I noticed that it doesn't seem to work correctly if you have multiple PCs from which to stream games.  If you switch from one PC to the other, the list of games from PC #1 persists when PC #2 is selected.  Otherwise, it looks really nice.

**Comment by Calcyfer** [comment]
@mattmeier612 Thanks, I'll try to reproduce and fix that.

**Comment by Calcyfer** [comment]
@mattmeier612 That seems to be behavior the current branch has, too. 
The list of games is kept in memory and is host-agnostic.

It's something I can fix, but it's beyond the scope of this PR.

File: index.html [hunk 1]
```diff
@@ -1,141 +1,174 @@
...
+                    <button type="button" class="mdl-button mdl-js-button mdl-button--raised mdl-button--colored mdl-js-ripple-effect" id="quitCurrentApp">
+                        <i class="material-icons">remove_circle_outline</i>
+                        Quit Current App
+                    </button>
+            </nav>
+        </div>
+    </header>
+    <main id="main-content" class="mdl-layout__content">
+            <div id="host-grid">
+                <div class="page-title">MoonLight Hosts</div>
```
Comment (cgutman) [inline index.html]: Please capitalize as `Moonlight` not `MoonLight`

**Comment by cgutman** [comment]
Merged as is. Will fixup the minor stuff myself.

**Comment by Calcyfer** [comment]
Cheers, give me a mention if there is anything I can fix / help with.

**Pull request merged**